src/org/jitsi/impl/neomedia/rtp/FrameDesc.java

```diff
@@ -182,6 +187,10 @@ public boolean lastSequenceNumberKnown()
     void setEnd(int end)
     {
         this.end = end;
+        if (this.end > this.maxSeen)
```

Comment:
Here we need to take sequence number wrap around into account. Alternatively, what we could do is unconditionally set the `maxSeen`, and prevent the `maxSeen` from being set if the `end` has been set.

However, I don't like adding this kind of logic in the setters. Maybe it'd be better to have a getMaxOrEnd() method.